examples/config files - basic/3 connector-ldap.yml

```diff
@@ -30,7 +30,7 @@ credential_manager:
     type: windows_credential_manager
     #service_name is Required for Windows Credential Manger
     service_name: "Internet or Network Address field in Credential Manager"
-    username: "UserName"
+    service_username: "UserName"
```

Comment:
Is there a really security concern of displaying username in plaintext? I demoed this to customer today and they did raise any concern on username part.

the problem with keyring is there is no method that will fetch the username.
get_password() required both service_name and username. This will return is password in a string.